std/net/curl.d

```diff
@@ -186,7 +186,8 @@ version(unittest)
     import std.file : deleteme;
     import std.path : buildPath;
 
-    import std.socket : Address, INADDR_LOOPBACK, Socket, TcpSocket;
+    import core.sys.windows.winsock2 : INADDR_LOOPBACK;
+    import std.socket : Address, Socket, TcpSocket;
```

Comment:
The tests fail on non-Windows platforms:
```
std/net/curl.d(189): Error: module core.sys.windows.winsock2 import 'INADDR_LOOPBACK' not found
```
The symbols in `core.sys.windows` are only defined when targeting Windows, with `version(Windows):` applied at the top.